MomsSpaghetti/Pages/Index.cshtml

```diff
@@ -225,15 +225,31 @@
 
                                 @foreach (var ingredient in Model.Recipe.Ingredients)
                                 {
+                                    @*<li class="recipe__item">
+                    <svg class="recipe__icon">
+                        <use href="img/icons.svg#icon-check"></use>
+                    </svg>
+                    <div class="recipe__count">2</div>
+                    <div class="recipe__ingredient">
+                        <span class="recipe__unit">g</span>
+                        pizza
+                    </div>
+                </li>*@
```

Comment:
For every ingredient in the recipe, this code will loop through the recipes and map the ingredients using @ingredient.ToString() in the index.cshtml file. It was initially hardcoded with 2, g and pizza.

Reply:
I'll confess I don't entirely understand this change, as I still see 2, g, and pizza.

Reply:
This is all part of the code that has been commented out.